dashboard/src/components/Config/AppRepoList/AppRepoList.v2.tsx

```diff
@@ -84,41 +130,53 @@ function AppRepoList({ cluster, namespace, kubeappsNamespace }: IAppRepoListProp
-                    { accessor: "url", Header: "URL" },
-                    { accessor: "accessLevel", Header: "Access Level" },
-                    { accessor: "namespace", Header: "Namespace" },
-                    { accessor: "actions", Header: "Actions" },
-                  ]}
-                  data={repos.map(repo => {
-                    return {
-                      name: repo.metadata.name,
-                      url: repo.spec?.url,
-                      accessLevel: repo.spec?.auth?.header ? "Private" : "Public",
-                      namespace: repo.metadata.namespace,
-                      actions: (
-                        <AppRepoControl
-                          repo={repo}
-                          secret={repoSecrets.find(secret =>
-                            secret.metadata.ownerReferences?.some(
-                              ownerRef => ownerRef.name === repo.metadata.name,
-                            ),
-                          )}
-                          namespace={namespace}
-                          kubeappsNamespace={kubeappsNamespace}
-                        />
-                      ),
-                    };
-                  })}
-                />
+                <h3>Global Repositories</h3>
+                <p>
+                  Global repositories are available for all Kubeapps users.{" "}
+                  {namespace !== kubeappsNamespace && (
+                    <>
+                      Administrators can go to the{" "}
+                      <Link to={app.config.apprepositories("default", kubeappsNamespace)}>
+                        {kubeappsNamespace}
+                      </Link>{" "}
+                      namespace to manage them.
+                    </>
+                  )}
+                </p>
+                {globalRepos.length ? (
+                  <Table
+                    valign="center"
+                    columns={tableColumns}
+                    data={getTableData(globalRepos, namespace !== kubeappsNamespace)}
+                  />
+                ) : (
+                  <p>No global repositories found.</p>
+                )}
+                {namespace !== kubeappsNamespace && (
+                  <>
+                    <h3>Namespace Repositories</h3>
+                    <p>
+                      Namespaced Repositories are only available in the current namespace. To change
+                      the namespace, use the "Current Context" selector in the top navigation.
+                    </p>
+                    {namespaceRepos.length ? (
+                      <Table
+                        valign="center"
+                        columns={tableColumns}
+                        data={getTableData(namespaceRepos, false)}
+                      />
+                    ) : (
+                      <p>
+                        The current namespace doesn't have any repository. Click on the button "Add
```

Comment:
"repository" should be plural here, but could also use "The { namespace } namespace doesn't have any repositories. ..." ?